### ISSUE TITLE
Can not able to upload new files on shared folder

### ISSUE THREAD

**Issue by Medow** [astrada/google-drive-ocamlfuse#596]
I am running google-drive-ocamlfuse version 0.7.15 on Ubuntu 18.04 server.
I am unable to add a new file on a folder shared with me as a contributor.
I can succefully edit and view and exist file and folder on that folder , but when ever i try to add a new file i got this message `Permission denied` .
Please advice me
Thanks,

**Comment by astrada** [comment]
Did you add the shared folder to your Drive (from the web interface)? Have you tried adding new files from the web interface? Do you get any error?

**Comment by Medow** [comment]
> Did you add the shared folder to your Drive (from the web interface)? Have you tried adding new files from the web interface? Do you get any error?

Yes, i can add new file to that folder successfully using rclone mounting and web interface.

**Comment by charly-esb** [comment]
Any new or recommendation in the matter? since google disabled the "add to my drive" option we are stuck with shortcuts which are interpreted as text files by google-drive-ocamlfuse.
Also .shared folder is mounted without writing permissions.
Thanks

**Comment by astrada** [comment]
> shortcuts which are interpreted as text files by google-drive-ocamlfuse

Which version are you running? Shortcut in the latest versions are mapped to symlinks.

**Comment by jtsauls** [comment]
I am using 0.7.22, but shortcuts from a folder in a shared drive in my personal drive are not symlinks in my mounted drive folder.

**Comment by astrada** [comment]
Well, shortcuts from My Drive to Shared Drives don't work because you can only mount a Drive at a time, so you would have a dangling shortcut that points to a non-mounted drive.

**Comment by shervinsahba** [comment]
@charly-esb Did using Shift+z to add the shared folder to your drive work? That is, were you able to write to a shared drive?

I'm able to add on the online platform, but it doesn't appear on the fuse drive. Only the symlinked versions appear.

**Comment by charly-esb** [comment]
> Well, shortcuts from My Drive to Shared Drives don't work because you can only mount a Drive at a time, so you would have a dangling shortcut that points to a non-mounted drive.

@astrada I have been able to mount two different drives using labels

**Comment by charly-esb** [comment]
> @charly-esb Did using Shift+z to add the shared folder to your drive work? That is, were you able to write to a shared drive?
> 
> I'm able to add on the online platform, but it doesn't appear on the fuse drive. Only the symlinked versions appear.

Totally, it is just as it is used to be.
maybe remount or clear cache would work?
remout:
fusermount -u /path/to/mount/dir
google-drive-ocamlfuse -label <label> /path/to/mount/dir
clear cache:
google-drive-ocamlfuse -label <label> -cc

Actually each time I make a major change I do all. Unmount, clear cache, mount.

**Comment by shervinsahba** [comment]
@charly-esb Thanks!
